### ISSUE TITLE
Dispatcher issue with latest code deployment on cloud

### ISSUE THREAD

**Issue by Gurjot-Singh1** [adobe/aem-guides-wknd#209]
### Expected Behaviour
New Image should be successfully Build and deployed on cloud via cloud Manager pipeline
### Actual Behaviour
Build & Unit Testing  is Success, but Unable to build new image.
#### Steps to Reproduce
Take Latest checkout from Git repo, 
do maven install on local for verification(Success)
Checkin the same code in your branch on cloud - https://git.cloudmanager.adobe.com/...
and Run the deployment Pipeline.

PS: 
Dispatcher validator fails inside Git repo but passes without any issues in clean folder (outside Git).
It is due to symlinks, for which I have enabled "symlinks = true" in config as well.

#### Platform and Version
Windows 10 
Deployment on Cloud Manager 

#### Logs taken while reproducing problem
[Log_msg.txt](https://github.com/adobe/aem-guides-wknd/files/6114434/Log_msg.txt)
[buildImage_log.log](https://github.com/adobe/aem-guides-wknd/files/6128472/buildImage_log.log)

**Comment by godanny86** [comment]
hi @Gurjot-Singh1 to confirm are you deploying to AEM as a Cloud Service?

**Comment by Gurjot-Singh1** [comment]
> hi @Gurjot-Singh1 to confirm are you deploying to AEM as a Cloud Service?

Yes, I am deploying to AEM as Cloud by checking in the code in a branch and then executing the Deployment pipeline.

**Comment by godanny86** [comment]
And you are working with the latest from the master branch? I'm assuming this must be related to Windows and the handling of symbolic links... Do you have an idea of the fix needed?

**Comment by Gurjot-Singh1** [comment]
Thanks @godanny86 for looking into this.
Yes, as mentioned I took the latest from Master branch and worked on it.
Also, for Symbolic links I just updated the "symlinks = true" in git config.
Is there any other setting/change required?
Using Windows 10 version 1909

**Comment by Gurjot-Singh1** [comment]
> And you are working with the latest from the master branch? I'm assuming this must be related to Windows and the handling of symbolic links... Do you have an idea of the fix needed?

Hi @godanny86  were you able to reproduce this?
or can you please point me into the right direction to make it work

**Comment by SAURABHsx** [comment]
Hi @Gurjot-Singh1 , Were you able to resolve this issue. I am also facing the same.
If you able to resolve, please share the steps.

**Comment by Gurjot-Singh1** [comment]
Hi

> Hi @Gurjot-Singh1 , Were you able to resolve this issue. I am also facing the same.
> If you able to resolve, please share the steps.

No, Saurabh. but as a work around I created a new project using acrheType 25 in local and pushed it on cloud. WKND is still not success.

**Comment by SAURABHsx** [comment]
> Hi
> 
> > Hi @Gurjot-Singh1 , Were you able to resolve this issue. I am also facing the same.
> > If you able to resolve, please share the steps.
> 
> No, Saurabh. but as a work around I created a new project using acrheType 25 in local and pushed it on cloud. WKND is still not success.

Thanks @Gurjot-Singh1

**Comment by SAURABHsx** [comment]
Hi @godanny86, Do you have any solution for this issue.

**Comment by Gurjot-Singh1** [comment]
Hi @godanny86,
Can you please guide us on this issue. as it is blocking us now.

**Comment by avitripathy01** [comment]
@Gurjot-Singh1 as a workaround, you could create the symbolic links using wsl in windows
